### PR TITLE
updated current contributors info and fixed small css issues

### PR DESCRIPTION
**Pull request by shrnkld**

### CODE REVIEW

File: docs/project/contributors.md [hunk 1]
```diff
@@ -30,26 +30,17 @@ Blocknet is an open-source project with a team consisting of contributors from a
...
-	<div class="platform">Discord <span class="handle">@michael#6510</span></div>
-</div>
-
-<div class="contributor">
-	<div class="fa fa-user-circle" aria-hidden="true"></div>
-	<div class="name">Kamel Bent Ahmed&nbsp;&nbsp;
-		<a href="https://www.linkedin.com/in/kamel-bent-ahmed-66211b13/" target="_blank"><i class="fa fa-linkedin-square" aria-hidden="true"></i></a>
+	<div class="img-round"><img src="/img/contributors/arlyn-culwick.jpg"></div>
+	<div class="name">Arlyn Culwick&nbsp;&nbsp;
+		<a href="https://www.linkedin.com/in/arlynculwick/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
```
Comment (synechist) [inline docs/project/contributors.md:40]: Replace with:
<a href="https://twitter.com/arlynculwick" target="_blank"><i class="fa fa-twitter-square fa-1x" aria-hidden="true"></i></a>

File: docs/project/contributors.md [hunk 2]
```diff
@@ -114,163 +80,49 @@ Blocknet is an open-source project with a team consisting of contributors from a
...
-		<a href="https://www.linkedin.com/in/kerst-bottema-436b4a26/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
-	</div>
-	<div class="title"><a href="https://block-node.info/" target="_blank">block-node.info</a></div>
-	<div class="platform">Discord <span class="handle">@ishkb1#0780</span></div>
-</div> -->
-
-<div class="contributor">
-	<div class="fa fa-user-circle" aria-hidden="true"></div>
-	<div class="name">@littleeagle&nbsp;&nbsp;
-		<a href="https://twitter.com/littleeagle20" target="_blank"><i class="fa fa-twitter-square fa-1x" aria-hidden="true"></i></a>
```
Comment (synechist) [inline docs/project/contributors.md:177]: Is littleeagle removing himself on account of his moderator role on r/satoshistreetbets?

File: docs/project/contributors.md [hunk 3]
```diff
@@ -114,163 +80,49 @@ Blocknet is an open-source project with a team consisting of contributors from a
...
-	<div class="platform">Discord <span class="handle">@xenix</span></div>
-</div> -->
+<div class="contributor"> 
+	<div class="img-round"><img src="/img/contributors/nikita-chisnikov.jpg"></div>
+	<div class="name">Nikita Chisnikov</div>
+	<div class="title">Bridge Bot</div>
+	<div class="platform">Discord <span class="handle">@nik_chis#7731</span></div>
+</div>
 
 </div>
```
Comment (synechist) [inline docs/project/contributors.md:127]: Do you think that the following people should be included? - 

Tryou
Coinwhisperer
Conan
Superbayfan
Amishbatman
Crypto-dev
Floyd1256
Uzrus

**Review by synechist — COMMENTED** [review]
One code update (link to Twitter instead of LinkedIn) and one suggestion to include several contributors.

File: docs/project/contributors.md (hunk 1 above)
Comment (synechist) [reply docs/project/contributors.md:40]: Hmm... the code wasn't included above.
Trying again:
<a href="https://twitter.com/arlynculwick" target="_blank"><i class="fa fa-twitter-square fa-1x" aria-hidden="true"></i></a>

Comment (synechist) [reply docs/project/contributors.md:40]: `<a href="https://twitter.com/arlynculwick" target="_blank"><i class="fa fa-twitter-square fa-1x" aria-hidden="true"></i></a>`

File: docs/project/contributors.md (hunk 3 above)
Comment (ConanMishler) [reply docs/project/contributors.md:127]: Good suggestions. Shorn & feel myself and Tryou should be added. Some of the others maybe a little later, after they've been with us a little longer and/or started submitting SB proposals, etc...

**Pull request merged**